desktop-environment.el

```diff
@@ -176,6 +193,17 @@ portion of the screen."
   "Shell command toggling bluetooth."
   :type 'string)
 
+
+;;; Helper variables - keyboard layout
+
+(defvar desktop-environment-keyboard-layouts-ring []
```

Comment:
please move this above the `defcustom` to avoid a compilation warning on the usage which comes before the definition.